### DISCUSSION TITLE
When will 3.0 be released?

### DISCUSSION THREAD

**Discussion by NebulaTechs** [filecxx/FileCentipede#644] [General]
It has been a long time since it was updated.

**Comment by filecxx** [comment]
I am currently working on another project which is not yet finished. If it goes well, I can start updating FileCentipede in October or November.

**Comment by NebulaTechs** [comment]
不过快一年了怎么还没动静。。。

**Reply by c-sanchez** [reply]
Well... considering that I have been translating the Spanish program for some time now and it has taken me a while, I understand, we have life and things that consume time without realizing it.

Besides, as they say, do you want it fast or do you want it right? 😆

**Comment by ZoinktheBoink** [comment]
Any updates, my guy? @filecxx 😘

**Reply by filecxx** [reply]
Soon.

**Reply by ZoinktheBoink** [reply]
WOO!!!!!!!!!!!
